docs/design.md

```diff
@@ -387,7 +387,7 @@ The back-end of TCMalloc has three jobs:
 
 *   It manages large chunks of unused memory.
 *   It is responsible for fetching memory from the OS when there is no suitably
-    sized memory available to fulfill an allocation requestion.
+    sized memory available to fulfil an allocation request.
```

Comment:
Small nit: I think both "fulfill" and "fulfil" are correct, https://www.merriam-webster.com/dictionary/fulfill

The two-l form looks a bit more common, https://books.google.com/ngrams/graph?content=fulfill%2Cfulfil&year_start=1800&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cfulfill%3B%2Cc0%3B.t1%3B%2Cfulfil%3B%2Cc0#t1%3B%2Cfulfill%3B%2Cc0%3B.t1%3B%2Cfulfil%3B%2Cc0

Reply:
Yeah, you're right. We can skip that one.